gamestonk_terminal/cryptocurrency/coingecko/pycoingecko_view.py

```diff
@@ -0,0 +1,1570 @@
+        print(e)
+        print("")
+        return None
+
+
+def view(coin: gecko_coin.Coin, other_args: List[str]):
+    """Plots chart for loaded cryptocurrency
+
+    Parameters
+    ----------
+    coin : gecko_coin.Coin
+        Cryptocurrency
+    other_args : List[str]
+        argparse arguments
+
+    """
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="chart",
+        description="""
+                        Display chart for loaded coin
+                        """,
+    )
+
+    parser.add_argument(
+        "--vs", default="usd", dest="vs", help="Currency to display vs coin"
+    )
+
+    parser.add_argument(
+        "-d", "--days", default=30, dest="days", help="Number of days to get data for"
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+
+        if not ns_parser:
+            return
+
+        df = coin.get_coin_market_chart(ns_parser.vs, ns_parser.days)
+        plt.figure(figsize=plot_autoscale(), dpi=PLOT_DPI)
+
+        plt.plot(df.index, df.price, "-ok", ms=2)
+        plt.xlabel("Time")
+        plt.xlim(df.index[0], df.index[-1])
+        plt.ylabel("Price")
+        plt.grid(b=True, which="major", color="#666666", linestyle="-")
+        plt.minorticks_on()
+        plt.grid(b=True, which="minor", color="#999999", linestyle="-", alpha=0.2)
+        plt.title(f"{coin.coin_symbol}/{df['currency'][0]}")
+        plt.show()
+        print("")
+
+    except SystemExit:
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def ta(coin: gecko_coin.Coin, other_args: List[str]):
+    """Load data for Technical Analysis
+
+    Parameters
+    ----------
+    coin : gecko_coin.Coin
+        Cryptocurrency
+    other_args : List[str]
+        argparse arguments
+
+    """
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="ta",
+        description="""
+                        Loads data for technical analysis
+                        """,
+    )
+
+    parser.add_argument(
+        "--vs", default="usd", dest="vs", help="Currency to display vs coin"
+    )
+
+    parser.add_argument(
+        "-d", "--days", default=30, dest="days", help="Number of days to get data for"
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+
+        if not ns_parser:
+            return
+
+        df = coin.get_coin_market_chart(ns_parser.vs, ns_parser.days)
+        return df
+
+    except SystemExit:
+        print("")
+        return None
+
+    except Exception as e:
+        print(e)
+        print("")
+        return None
+
+
+def base_info(coin: gecko_coin.Coin, other_args: List[str]):
+    """Shows basic information about loaded coin
+
+    Parameters
+    ----------
+    coin : gecko_coin.Coin
+        Cryptocurrency
+    other_args : List[str]
+        argparse arguments
+
+    """
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="base_info",
```

Comment:
name of program should be the command used by the user, which in this case is just "info"